composer.json

```diff
@@ -16,7 +16,8 @@
         "gitter": "https://gitter.im/hybridauth/hybridauth"
     },
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "firebase/php-jwt": "^5.0"
```

Comment:
@miled @nitipatl

I created nitipatl/hybridauth#1 where `JWT` is moved to third-party packages of Hybridauth. 
Is this OK. I think it is because `JWT` will maybe be also used by other packages in the future.

So, @nitipatl should merge that PR, so it will be added to this.